roles/postgresql_upgrade/tasks/main.yml

```diff
@@ -0,0 +1,58 @@
+---
+- name: Get version of postgres data directory
+  slurp:
+    path: /var/lib/pgsql/data/PG_VERSION
+  register: pg_version
+
+- when:
+    - ansible_distribution_major_version == '8'
```

Comment:
This should also check for this being *RHEL* 8, not Debian 8 (even if we don't support Debian 8 anymore)

Plus I'd put this check as the first task, `/var/lib/pgsql` won't exist on EL7 installs with SCL'ed PostgreSQL anymore.